Libraries/ReactNative/requireNativeComponent.windows.js

```diff
@@ -0,0 +1,233 @@
+/**
```

Comment:
do we need to maintain a copy of this versus referencing it from node_modules/react-native somehow? I don't see any source deviations.

Reply:
@matthargett - I don't think this change will land in react-native until 0.56, so we need to keep it for RNW 0.55 - then can delete.

Reply:
maybe @grabbou can import it into 55.1 since it doesn't chance the behavior?

Reply:
maybe - but I have this workaround so I'm not in a rush - I hate to ask @grabbou to cherry-pick when I've already made him do so much extra work for the link bugs 🤦‍♂️

Reply:
I'd prefer to avoid cherry-picking when possible. Our latest works with @rozele did a few side issues post-release which led to unwanted bumps back in 0.52.x branch.